### ISSUE TITLE
Custom staff not adding highest spell level entry

### ISSUE THREAD

**Issue by jjnich** [jessev14/pf2e-staves#10]
With the following description, only cantrips and lvl 1 spells are added to the spellcasting entry:
```
This long piece of wood can aid in walking and deliver a mighty blow. 

Cantrips: @UUID[Compendium.pf2e.spells-srd.gISYsBFby1TiXfBt]{Acid Splash} @UUID[Compendium.pf2e.spells-srd.gpzpAAAJ1Lza2JVl]{Detect Magic}

1st: @UUID[Compendium.pf2e.spells-srd.TTwOKGqmZeKSyNMH]{Feather Fall}

2nd: @UUID[Compendium.pf2e.spells-srd.f8hRqLJaxBVhF1u0]{Acid Arrow}
```

But when I edit the staff description to the following, cantrips, 1st and 2nd level spells are added to the spellcasting entry:
```
This long piece of wood can aid in walking and deliver a mighty blow. 

Cantrips: @UUID[Compendium.pf2e.spells-srd.gISYsBFby1TiXfBt]{Acid Splash} @UUID[Compendium.pf2e.spells-srd.gpzpAAAJ1Lza2JVl]{Detect Magic}

1st: @UUID[Compendium.pf2e.spells-srd.TTwOKGqmZeKSyNMH]{Feather Fall}

2nd: @UUID[Compendium.pf2e.spells-srd.f8hRqLJaxBVhF1u0]{Acid Arrow}

3rd:
```

It seems like the way the array of spells is being referenced is off by 1

**Comment by jjnich** [comment]
I've found if I add another line to the description (like "Craft Requirements") it seems to work as expected. Perhaps it's just not reading the last line of the description because there's never spells there on the default staves.

**Comment by jessev14** [comment]
It's definitely supposed to read the entire description. I'll take a look.

**Comment by jessev14** [comment]
Should be fixed in version 1.1.5.
